src/metabase/sync/sync_metadata/fields.clj

```diff
@@ -102,12 +143,14 @@
         {new-database-type :database-type, new-base-type :base-type} field-metadata]
     ;; If the driver is reporting a different `database-type` than what we have recorded in the DB, update it
     (when-not (= old-database-type new-database-type)
+      (mark! update-database-type-count)
```

Comment:
Updating a database type seems like it should be a rare occurrence. Added this as it might be something of interest.